forge-gui/res/cardsfolder/upcoming/urza_chief_artificer.txt

```diff
@@ -2,7 +2,7 @@ Name:Urza, Chief Artificer
 ManaCost:3 W U B
 Types:Legendary Creature Human Artificer
 PT:4/5
-K:Affinity:Creature.Artifact+YouCtrl:artifact creatures
+K:Affinity:Creature.Artifact
```

Comment:
should be good with `K:Affinity:Creature.Artifact:artifact creature`

YouCtrl is added automatically in CardFactoryUtil.addStaticAbility()

@Hanmac we did have to update the description stuff for Junk Winder, so I think this should be good